### ISSUE TITLE
Centos install issues

### ISSUE THREAD

**Issue by garetht2** [duo-labs/cloudmapper#57]
I had some install pains on a plain, fresh built centOS7 on AWS:

normal instructions:
# clone the repo
git clone git@github.com:duo-labs/cloudmapper.git
# (Centos, Fedora, RedHat etc.):
sudo yum install autoconf automake libtool python-dev jq
cd cloudmapper/

at this point I had to run:
curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py"
pip install virtualenv

then I could continue 
virtualenv venv
source venv/bin/activate
pip install -r requirements.txt

then it bombed out with a python error, so I had to run
yum install python-devel

In summary, I had to run the additional commands at any time before the command "virtualenv venv"
curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py"
pip install virtualenv
yum install python-devel

**Comment by 0xdabbad00** [comment]
Makes sense, my instructions assumed `pip` and `virtualenv` were already installed, which is rarely the case in a new setup.  I'll add some guidance around that.

**Comment by garetht2** [comment]
Also (stupid centos) for the collect_data.sh script to work I had to run:

yum install epel-release
yum install jq

**Comment by garetht2** [comment]
For the python cloudmapper.py script to work I had to run:

pip install pyjq
pip install netaddr


I'm determined to get this to work!  :)

**Comment by garetht2** [comment]
It's beautiful!

Thank you.

**Comment by asloan7** [comment]
Yes, on Centos it seems there is no python-dev, only python-devel https://centos.pkgs.org/7/centos-x86_64/python-devel-2.7.5-58.el7.x86_64.rpm.html

**Comment by 0xdabbad00** [comment]
Thank you. Resolved in https://github.com/duo-labs/cloudmapper/commit/11565c9171f6d086eb6fabbbe44f7074a93c725c
